ci-operator/config/openshift/azure-file-csi-driver/openshift-azure-file-csi-driver-release-4.19.yaml

```diff
@@ -76,6 +76,13 @@ tests:
   steps:
     cluster_profile: azure4
     workflow: openshift-e2e-azure-csi-file-nfs
+- as: e2e-azure-csi-file-cross-subscription
```

Comment:
We only add it for 4.19 file csi driver presubmit job? Better to add it to payload periodic(weekly/monthly) job both 4.19 and 4.20.

Reply:
Good catch, yes I should add it in master branch in my origin PR.

Reply:
Not sure this is a good fit for a payload/periodic, and since this requires users to create their own storageclass it does not seem likely to break with operator change. But curious what others think though. Otherwise looks good to me!

Reply:
@Phaow PR was merged, we can add periodic CI in seperate PR if we really need it.

Reply:
Sure thing, agree since operator changes impact this in low probability, we could keep current status.